gustaf/io/mfem.py

```
@@ -107,97 +107,176 @@ def extract_values(fname, start_index, n_lines, total_lines, dtype):
+    boundary_string += f"{boundary_array_string}\n\n"
+
+    # Vertices
+    vdim = 2  # Currently only option
+    vertices_array_string = format_array(mesh.vertices)
+    nvertices = mesh.vertices.shape[0]
+    vertices_string = f"vertices\n{nvertices}\n{vdim}\n"
+    vertices_string += f"{vertices_array_string}"
+    return elements_string, boundary_string, vertices_string
+
+
+def _export_3d(mesh):
```

Ruff (0.14.3)

Missing return type annotation for private function `_export_3d`

(ANN202)